Homework_7_Java_Arrays_Loops/JavaArraysAndLoops/src/StudentSearch.java

```diff
@@ -0,0 +1,50 @@
+public class StudentSearch {
+    /*
+    Опис задачі:
+У вашому списку студентів ви хочете знайти конкретного студента за його номером залікової книжки.
+Напишіть програму, яка використовує цикл for each для перегляду списку студентів і знаходить номер залікової книжки,
+який ви шукаєте. Якщо студент знайдений, програма повинна вивести повідомлення і завершити роботу.
+Якщо студент не знайдений, програма повинна вивести відповідне повідомлення і завершити роботу.
+Вхідні дані:int[] studentIDs = {12345, 67890, 54321, 98765, 24680};
+
+int searchID = 54321;
+Очікуваний результат: у разі, якщо номер залікової книжки був знайдений у списку - вивести повідомлення про це і завершити роботу -
+якщо такої заліковки немає у списку - вивести відповідне повідомлення і завершити роботу
+     */
+
+    //for each
+
+    public static void main(String[] args) {
+        int[] studentIDs = {12345, 67890, 54321, 98765, 24680};
+        int searchID = 54321;
+        boolean isFound = false;
+
+        for (int i : studentIDs) {
```

Comment:
valid, but better practice would be to call it smth like `studentId` which would better represent what it holds inside it during the iteration

Reply:
thanks